taskiq/result_backends/redis/redis_client.py

```diff
@@ -0,0 +1,72 @@
+from typing import Mapping, Optional, Union
+
+from errors import RedisConnectionError
+from redis.asyncio import ConnectionPool, Redis
+
+
+class RedisClient:
```

Comment:
In my opinion the RedisClient class is redundant. You can use raw pool in RedisAsyncResultBackend.